### PR TITLE
Internal: Release/3.9.0

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: core/kits/documents/tabs/theme-style-form-fields.php [hunk 1]
```diff
@@ -136,16 +136,14 @@ protected function register_tab_controls() {
...
 			'form_field_focus_transition_duration',
 			[
-				'label' => esc_html__( 'Transition Duration', 'elementor' ) . ' (ms)',
+				'label' => esc_html__( 'Transition Duration', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'selectors' => [
-					$input_selector => 'transition: {{SIZE}}ms',
+				'size_units' => [ 's', 'ms' ],
+				'default' => [
+					'unit' => 'ms',
```
Comment (matipojo) [inline core/kits/documents/tabs/theme-style-form-fields.php:143]: @rami-elementor BC? What will happen for the old setting?

File: core/kits/documents/tabs/theme-style-images.php [hunk 2]
```diff
@@ -200,16 +200,14 @@ protected function register_tab_controls() {
...
+				'label' => esc_html__( 'Transition Duration', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'range' => [
-					'px' => [
-						'max' => 3,
-						'step' => 0.1,
-					],
+				'size_units' => [ 's', 'ms' ],
+				'default' => [
+					'unit' => 's',
```
Comment (matipojo) [inline core/kits/documents/tabs/theme-style-images.php:207]: @rami-elementor BC?

File: includes/widgets/video.php [hunk 3]
```diff
@@ -1084,6 +1084,10 @@ public function get_embed_params() {
 			$params['color'] = str_replace( '#', '', $settings['color'] );
 
 			$params['autopause'] = '0';
+
+			if ( ! empty( $settings['yt_privacy'] ) ) {
+				$params['dnt'] = 'true';
+			}
```
Comment (matipojo) [inline includes/widgets/video.php:1090]: @KingYes Maybe it should be here:
https://github.com/elementor/elementor/blob/master/assets/dev/js/frontend/handlers/video.js#L114

File: core/kits/documents/tabs/theme-style-form-fields.php (hunk 1 above)
Comment (rami-elementor) [reply core/kits/documents/tabs/theme-style-form-fields.php:143]: The old selector had hard-coded unit `{{SIZE}}ms`. For BC, I replaced the selector with `{{SIZE}}{{UNIT}}` and added a default of `'unit' => 'ms'`.

File: core/kits/documents/tabs/theme-style-images.php (hunk 2 above)
Comment (rami-elementor) [reply core/kits/documents/tabs/theme-style-images.php:207]: The old selector had hard-coded unit `{{SIZE}}s`. For BC, I replaced the selector with `{{SIZE}}{{UNIT}}` and added a default of `'unit' => 's'`.

**Pull request closed without merging**